apps/trading/e2e/tests/deal_ticket/test_isolated_cross_margin.py

```diff
@@ -51,3 +57,47 @@ def test_switch_cross_isolated_margin(
     next_epoch(vega=vega)
     expect(page.locator(margin_row).nth(1)).to_have_text(
         "22,109.99996Cross1.0x")
+
+
+@pytest.mark.usefixtures("auth", "risk_accepted")
+def test_check_cross_isolated_margin_info(
+        continuous_market, vega: VegaServiceNull, page: Page):
+    create_position(vega, continuous_market)
```

Comment:
Dumb question, but why use a seperate function for this create position?

Reply:
an open position is needed to be able to switch margin mode

Reply:
Sorry what i mean is, why is this a seperate function rather than the code being in the test. The function is only used once?

Reply:
both tests test_switch_cross_isolated_margin and test_check_cross_isolated_margin_info use this and this is more readable in a separate function